releng/build_codecs.sh

```diff
@@ -72,7 +72,9 @@ download_check_extract_pushd() {
         exit 1
       fi
     fi
+    set +e # Allow this to exit with non-zero for win32 strangeness
```

Comment:
I've put in this slightly horrible workaround for now.
I had a look at a newer windows image as suggested but it seemed to be missing a lot of build dependencies.

Reply:
It will do, thanks!